File: src_python/03_akar_kuadrat.py
a = float(input("Masukkan nilai a: "))
b = float(input("Masukkan nilai b: "))
c = float(input("Masukkan nilai c: "))

D = b**2 - 4*a*c

if D < 0:
    print("Persamaan hanya memiliki akar imajiner")
else:
    x1 = (-b + math.sqrt(D)) / (2*a)
    x2 = (-b - math.sqrt(D)) / (2*a)

    print(f"Akar 1 = {x1:.3f}")
    print(f"Akar 2 = {x2:.3f}")

File: src_R/03_akar_kuadrat.R
a <- 2
b <- 4
c <-5

D <- b^2 - 4*a*c

if (D < 0) {
  cat("Akar imajiner\n")
} else {
  x1 <- (-b + sqrt(D)) / (2*a)
  x2 <- (-b - sqrt(D)) / (2*a)
  
  cat(sprintf("Akar 1 = %.3f\n", x1))
  cat(sprintf("Akar 2 = %.3f\n", x2))
}
